reference/onenote/notebook.md

```diff
@@ -119,6 +121,37 @@ OneNote.run(function (context) {
 }); 
 ```
 
+### getRestApiId()
+Gets the id that is compatible with the REST API.
+
+#### Syntax
+```js
+notebookObject.getRestApiId();
+```
+
+#### Parameters
+None
+
+#### Returns
+string
+
+#### Examples
+```js
+
+OneNote.run(function(ctx){
+    // Get the current notebook.         
+    var notebook = ctx.application.getActiveNotebook();
+    var restApiId = notebook.getRestApiId();
+
+    return ctx.sync().
+        then(function(){
+            console.log("The rest api id is " + restApiId.value);
+            // Note that the rest API id isn't all you need to interact with the OneNote Rest API. For SharePoint notebooks, the notebook baseUrl should be used to talk to OneNote REST APIs according to https://blogs.msdn.microsoft.com/onenotedev/2015/06/11/and-sharepoint-makes-three/
```

Comment:
Change "rest API id" to "REST API ID", and change "OneNote Rest API" to "OneNote REST API". Also, change the URL to a link - [OneNote development blog](https://blogs.msdn.microsoft.com/onenotedev/2015/06/11/and-sharepoint-makes-three/). (The title of that blog post isn't super clear that it applies to OneNote.)